modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -1570,7 +1570,7 @@ and should be below ~3.0 pixels.
 
 @param image Gray image used to find chessboard corners
 @param patternSize Size of a found chessboard pattern
-@param corners Corners found by findChessboardCorners(SB)
```

Comment:
Please confirm this correction before merging

Reply:
Documentation build is available here: http://pullrequest.opencv.org/buildbot/export/pr/20235/docs/
(artifact of "Docs" builder)

Reply:
findChessboardCorners(SB) was corrected to findChessboardCornersSB(). 

This is under the assumption that findChessboardCorners(SB) was a typo, given no such function exits.